frontend/src/concepts/k8s/NameDescriptionField.tsx

```diff
@@ -22,6 +27,7 @@ const NameDescriptionField: React.FC<NameDescriptionFieldProps> = ({
   autoFocusName,
   showK8sName,
   disableK8sName,
+  validNotebookName,
```

Comment:
`NameDescriptionField` is a generic component and therefore `validNotebookName` is too specific for notebooks.

Perhaps use props `isNameValid` and `nameHelperText` instead